test/functional/api/command_spec.lua

```diff
@@ -782,6 +782,10 @@ describe('nvim_create_user_command', function()
     eq(5, #api.nvim_list_tabpages())
     eq(1, fn.tabpagenr())
   end)
+  it('addr option allows ranges without explicit range option', function()
+    feed([[:call nvim_create_user_command('Test2', 'echo "hello"', #{addr: 'other'})<CR>]])
```

Comment:
should we test that the lines actually got received?
```suggestion
    feed([[:call nvim_create_user_command('Test2', 'echo "hello <line1><line2>"', #{addr: 'other'})<CR>]])
```